docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+.. post:: March 19, 2018
+   :tags: portland-2018, volunteers, shirts, tickets
+
+Call for Volunteers, Shirts on Sale, Tickets Update
+===================================================
+
+Greetings, fellow documentarians! The Portland conference is just short of two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
```

Comment:
I read this as "just two short months away", perhaps "less than two months away"